pom.xml

```diff
@@ -16,20 +16,20 @@
   <properties>
     <maven.compiler.source>21</maven.compiler.source>
     <maven.compiler.target>21</maven.compiler.target>
-    <spring.boot.version>3.2.5</spring.boot.version>
-    <jib-maven-plugin.version>3.4.2</jib-maven-plugin.version>
-    <test-containers.version>1.19.7</test-containers.version>
-    <maven-surefire.version>3.2.5</maven-surefire.version>
+    <spring.boot.version>3.3.3</spring.boot.version>
+    <jib-maven-plugin.version>3.4.3</jib-maven-plugin.version>
```

Comment:
Does this work? Chris had an fatal error when building the image using v3.4.3

Reply:
It doesn't seem to work, no. I have reverted the version.